www/package.json

```diff
@@ -148,6 +148,9 @@
     "raf": "~3.3.2",
     "react-hot-loader": "^4.2.0",
     "react-test-renderer": "^16.8.0",
+    "react-virtualized": "^9.21.2",
```

Comment:
What are these being used for?

Reply:
I think they are used for '/client/examples/EventOverlay/KitchenSink.js'
When I run 'yarn start'. It came out the error: Module not found. Can't resolve 'react-virtualized', 'react-virtualized-auto-sizer', 'react-window'

Reply:
ok